janus_core/helpers/mlip_calculators.py

```diff
@@ -146,6 +205,19 @@ def choose_calculator(
             model = model if model else "small"
             kwargs.setdefault("default_dtype", "float64")
 
+            # Remove dispersion (to be added later for any calculator)
+            if dispersion and not kwargs.get("dispersion", True):
```

Comment:
`dispersion` is an argument and thus can't be in `kwargs`?

Reply:
Good point! That simplifies things...